src/Client/RemoteRepoFileFetcherInterface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Tuf\Client;
+
+/**
+ * Defines an interface for fetching remote repo files.
+ */
+interface RemoteRepoFileFetcherInterface
```

Comment:
Why not just `RepoFileFetcherInterface`? Couldn't we implement the fetching against local data sources equivalently, even just for testing?

Reply:
Sounds good. Changed the name.